### DISCUSSION TITLE
Does it collect the metrics from different node in parallel ?

### DISCUSSION THREAD

**Discussion by tutaidalal** [jacksontj/promxy#628] [Q&A]
Does it collect the metrics from different node in parallel ? if not where is the code to enhance to call nodes in parallel ?

**Comment by jacksontj** [comment]
Yes, all calls to all downstream servergroups and nodes are made in parallel. This is how promxy can maintain its [generally quick performance](https://github.com/jacksontj/promxy#what-is-query-performance-like-with-promxy).

**Reply by tutaidalal** [reply]
Thanks for the update.
